_includes/_render_nav.html

```diff
@@ -1,3 +1,27 @@
 {% for item in include.nav %}
-	<li><a href="{{item.url}}">{{item.title}}</a></li>
+
+	{% if item.sub !=null %}
+		<div class="dropdown">
+
+			<a class="arrow">{{item.title}}</a>
+
+			<ul class="submenu">
+
+	  		{% for subitem in item.sub %}
+	  			<li><a href="{{subitem.url}}">{{subitem.title}}</a></li>
+	  		{% endfor %}
+
+			</ul>
+		</div>
+
+  	{% else %}
+	  	{% if item.icon %}
+			<a class="{{item.icon}}" href="{{item.url}}"></a>
+
+	  	{% else %}
+			<a href="{{item.url}}">{{item.title}}</a>
+
+		{% endif %}
+
+	{% endif %}
 {% endfor %}
```

Comment:
Also here, can we insert a new line?
I think this file also need a nice formatting. Can you please check that the indentation is done with 2 spaces and no 4 tabs? And that there are no random empty lines or weird indentation. Thanks :)